docs/iris/src/copyright.rst

```diff
@@ -2,34 +2,21 @@
 Iris copyright, licensing and contributors
 ==========================================
 
-.. |copy|   unicode:: U+000A9 .. COPYRIGHT SIGN
-
 Iris code
 ---------
 
-All Iris source code, unless explicitly stated, is |copy| ``British Crown copyright, 2014`` and 
-is licensed under the **GNU Lesser General Public License** as published by the
-Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+All Iris source code, unless explicitly stated, is ``Copyright Iris
+Contributors`` and is licensed under the **GNU Lesser General Public
+License** as published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
 You should find all source files with the following header:
 
 .. admonition:: Code License
 
-    |copy| British Crown Copyright 2010 - 2014, Met Office
-    
-    This file is part of Iris.
-    
-    Iris is free software: you can redistribute it and/or modify it under
-    the terms of the GNU Lesser General Public License as published by the
-    Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-    
-    Iris is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU Lesser General Public License for more details.
+    Copyright Iris Contributors
     
-    You should have received a copy of the GNU Lesser General Public License
-    along with Iris.  If not, see `<http://www.gnu.org/licenses/>`_. 
+    This file is part of Iris and is released under the LGPL license.
+    See LICENSE in the root of the repository for full licensing details.
```

Comment:
`See COPYING and COPYING.LESSER in the root of the repository for full licensing details.`

Reply:
👍